app/components/c-attributegrid.vue

```diff
@@ -0,0 +1,247 @@
+<template>
+  <div :class="b(modifiers)">
+    <div :class="b('table', { open: isOpen })" :style="{ 'max-height': tableMaxHeight }">
+      <div
+        v-for="(attribute, index) in attributes"
```

Comment:
- [ ] Please update template according to new attribute data structure from backend:

```
"tech_attributes" : [
	{
		"key": "someKey",
		"value": "someValue",
		"url": "URL for search || empty"
	},
	{}
]
```

Reply:
Note: discussion with BE if we should also include an emphasis flag is still open. (so we don't have to guess which item we should make bold)